plasmapy/physics/dielectric.py

```diff
@@ -16,6 +16,8 @@
            'cold_plasma_permittivity_LRP',
            'permittivity_1D_Maxwellian']
 
+StixTensorElements = namedtuple("Result1", ["sum", "difference", "plasma"], )
+RotatingTensorElements = namedtuple("Result1", ["left", "right", "plasma"], )
```

Comment:
Would this perhaps be more descriptive?
```Python
StixTensorElements = namedtuple("StixTensorElements", ["sum", "difference", "plasma"], )
RotatingTensorElements = namedtuple("RotatingTensorElements", ["left", "right", "plasma"], )
```

Reply:
Gah! Of course I would forget to change those.

Reply:
Addressed by afa8919